dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/ShutdownServiceImpl.java

```diff
@@ -160,13 +162,14 @@ public Object call() {
                     // against a delta since doubles can sometimes report back values close to 1.0, but not
                     // exactly 1.0
                     final double difference = progress.get() - 1.0;
-                    if (difference > 0.0001 || difference < -0.0001) {
-                        job.cancel();
-                        ds3Client.cancelJobSpectraS3(new CancelJobSpectraS3Request(jobId));
+                        if (difference > 0.0001 || difference < -0.0001) {
+                            if (job instanceof JobTask) {
```

Comment:
Would it be an error if this wasn't an instance of JobTask?

Reply:
On the other hand we don't care about waiting for a task that isn't a Job Task